src/components/ArticleList.jsx

```diff
@@ -6,14 +6,17 @@ import { useParams } from "react-router-dom";
 const ArticleList = () => {
   let { topic } = useParams();
 
+  const [isLoading, setIsLoading] = useState(true);
+
   const [articles, setArticles] = useState([]);
 
   useEffect(() => {
     getArticles(topic).then(({ articles }) => {
       setArticles(articles);
+      setTimeout(() => setIsLoading(false), 2000);
     });
   }, [topic]);
-
+  if (isLoading) return <h1>Loading...</h1>;
```

Comment:
sorry to be nit-picky! 😅

try to limit your uses of `<h1>` tags!
There should only be one per html page

You already have one in your `Header`